packages/lib-classifier/src/plugins/drawingTools/models/marks/Mark/Mark.js

```diff
@@ -102,6 +102,10 @@ const BaseMark = types
       self.finished = true
     },
 
+    continueDrawing() {
+      self.finished = false
+    },
+
```

Comment:
❓ [major] Should this be in the base Mark model, or the Polygon Mark model? I think it should be shifted down to the Polygon Mark, right?

Reply:
I'm thinking it should stay in `Mark` model because `finished` is created and initialized in the `Mark` model. Also, keeping it in the `Mark` model will future-proof this for any other tools that may need it.
Thoughts?

Reply:
Future-proofing is a good point, actually.

I still feel that `continueDrawing` is something that should go in the `Polygon` mark (and any future Polygon-like mark) though, since `mark.continueDrawing()` is _only_ called by the `Polygon` component (and any future Polygon-like component) and not by the InteractionLayer nor anything _shared by all types of Marks.

On a scale of 0 (ambivalent) to 10 (pistols at dawn, good sir, I will duel to defend my opinion), I feel like I'm a... 5 (I will tweet angrily but won't actually do anything #continueDrawingInPolygon) on this.

If you feel more strongly than I do, then keep is as is - all I ask is that you update the Mark's section in the plugins/drawingTools/README accordingly. 👌